dev/NavigationView/NavigationViewItem.h

```diff
@@ -36,7 +36,6 @@ class NavigationViewItem :
     // IControlOverrides overrides
     void OnGotFocus(winrt::RoutedEventArgs const& e) override;
     void OnLostFocus(winrt::RoutedEventArgs const& e) override;
-    void OnKeyDown(winrt::KeyRoutedEventArgs const& args);
```

Comment:
>OnKeyDown [](start = 9, length = 9)

Where was this defined before? I can't find where it was deleted.

Reply:
There was no function definition before, so I just removed it.